app/components/charts/item-location-sales-trend/template.hbs

```diff
@@ -1,6 +1,9 @@
 {{#ui/loading-panel isLoading=isLoading}}
-  {{ui/title-bar title="Sales trends"}}
-  <div class="chart">
+  {{ui/title-bar
+    title=(concat "Sales trends" (if isHidden " +" " -"))
+    click=(action "toggleOpenClose")
```

Comment:
You could possibly add a `toggleSetting` action on the `prefsService` with a clojure action: `(action preferencesService.toggleSetting "itemLocationSalesTrendOpen")`